jekyll/_cci2/server/installation/1-install-prerequisites.adoc

```diff
@@ -0,0 +1,691 @@
+---
```

Comment:
This doc confuses me w.r.t. how it relates to this other phase 1 prereq doc https://github.com/circleci/circleci-docs/blob/156776f0d0b5eff210db93d1144e3c9791d05b7b/jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

Reply:
i think it may have been deleted after i created this branch. thanks for the call out